javascripts/developer-materials.angular.js

```diff
@@ -314,8 +575,7 @@ dcp.controller('developerMaterialsController', function($scope, materialService)
     // $scope.paginate.pagesArray = new Array(pages);
     $scope.paginate.currentPage = page;
 
-    // $scope.data.displayedMaterials = [];
-    $scope.data.displayedMaterials = $scope.data.materials.slice(startAt,endAt);
+    $scope.data.displayedMaterials = $scope.data.materials;
```

Comment:
Could you let me know how you are handling pagination now? If you are just reassigning the displayedMaterials, I don't think we need this bit, could just use `$scope.data.materials` directly.